modules/swffg-main.js

```diff
@@ -586,6 +586,21 @@ function isCurrentVersionNullOrBlank(currentVersion) {
 Hooks.once("ready", async () => {
   SettingsHelpers.readyLevelSetting();
 
+  if (!game.settings.get("starwarsffg", "token_configured")) {
+    const tokenData = {
+      bar1: {
+        attribute: "stats.wounds",
+      },
+      bar2: {
+        attribute: "stats.strain",
+      },
+      displayBars: 30, // hovered by anyone
+    };
+    game.settings.set("core", "defaultToken", tokenData);
```

Comment:
Do we not need to merge `tokenData` with whatever the current value of the `defaultToken` core setting is here? Or will it handle that automatically?

Reply:
It does need to be merged, good catch. Updated with a new commit.